### PR TITLE
Aud 7777

### PR DESCRIPTION
**Pull request by arantzavillicana**
Login faliido gg

### CODE REVIEW

File: Prueba.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Prueba :D
```
Comment (fernando-cbqa) [inline Prueba.txt]: delete this file pls

File: README - Copy.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Training - Web Project
+
```
Comment (fernando-cbqa) [inline README - Copy.md]: delete this file pls

File: Wenasoy.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Training - Web Project
+
```
Comment (fernando-cbqa) [inline Wenasoy.md]: delete this file pls

File: src/main/java/pageObject/Login_page.java [hunk 4]
```diff
@@ -29,6 +29,14 @@ public void successLogin(String Alias, String Email,String Pass){
         Assert.assertEquals(CurrentUrl,"https://app.auditate.mx/panel/principal");
     }
 
+    public void failedLogin(String Alias, String Email,String Pass){
+        driver.findElement(By.cssSelector("input[placeholder='Alias de la Empresa']")).sendKeys(Alias);
+        driver.findElement(By.cssSelector("input[placeholder='Correo Electrónico']")).sendKeys(Email);
+        driver.findElement(By.cssSelector("input[placeholder='Contraseña']")).sendKeys(Pass);
+        driver.findElement(By.cssSelector("button[type=submit]")).click();
+        //Validation to login failed
+        driver.findElement(By.cssSelector("div[role='dialog']")).isDisplayed();
```
Comment (fernando-cbqa) [inline src/main/java/pageObject/Login_page.java]: validate the next texts:
image

**Comment by arantzavillicana** [comment]
aiuda, no puedo quitar los files que no me sirveeeen

**Review by fernando-cbqa — APPROVED** [review]
looks good

File: Prueba.txt (hunk 1 above)
Comment (arantzavillicana) [reply Prueba.txt]: Done

File: README - Copy.md (hunk 2 above)
Comment (arantzavillicana) [reply README - Copy.md]: Done

File: Wenasoy.md (hunk 3 above)
Comment (arantzavillicana) [reply Wenasoy.md]: Done

File: src/main/java/pageObject/Login_page.java (hunk 4 above)
Comment (arantzavillicana) [reply src/main/java/pageObject/Login_page.java]: Done

**Review by Andrea-CBQA — APPROVED** [review]
Everything looks good :)

**Pull request merged**